data/options.nix

```diff
@@ -62,6 +62,17 @@ keyOptionsType = submodule ({ ... }: {
         prior to moving the secret in place.
       '';
     };
+
+    uploadAt = mkOption {
+      default = "pre-activation";
+      type = enum [ "pre-activation" "post-activation" ];
```

Comment:
Should we add a third type of `manual` that requires one to use `morph upload-secrets` instead?

Reply:
I don't think we should until someone finds an actual need for it, and even then we should consider whether there's another deficiency before adding it. Thanks for giving it thought though. :)